resources/public/api/conf/2.0/testdata/fund-release-outcome.md

```diff
@@ -279,7 +281,8 @@
         <pre class="code--block">
 {
   "code": "FUND_RELEASE_SUPERSEDED",
-  "message": "This fund release has already been superseded"
+  "message": "This fund release has already been superseded",
+  "lifeEventId": "3456789001"
```

Comment:
As with extensions - this change is correct, but the fund release id in the request needs updating to be 3456789000